### PR TITLE
Codechef TACHSTCK Solution in C

### PR DESCRIPTION
**Pull request by ritish099**
**Fixes issue:#4969**

**Changes:**

Added Codechef TACHSTCK solution in C
Added README for Codechef TACHSTCK problem

### CODE REVIEW

**Comment by ritish099** [comment]
Sir @arnavb , Check it out

File: code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#include<stdio.h>
+
```
Comment (arnavb) [inline code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c]:
```suggestion
```

File: code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            continue;
+        else 
+		{
+            count += 1;
+            i -= 1;
+        }
+    }
+    printf("%lld\n", count);
+    return 0;
+
```
Comment (arnavb) [inline code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c]:
```suggestion
```

File: code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c [hunk 3]
```diff
@@ -0,0 +1,80 @@
+#include<stdio.h>
+
+#include<stdlib.h>
```
Comment (arnavb) [inline code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c]:
```suggestion
#include <stdlib.h>
```

File: code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c [hunk 4]
```diff
@@ -0,0 +1,80 @@
+#include<stdio.h>
```
Comment (arnavb) [inline code/online_challenges/src/codechef/TACHSTCK/TACHSTCK.c]:
```suggestion
#include <stdio.h>
```

**Comment by ritish099** [comment]
Sir @arnavb , Check it out

**Pull request merged**